### DISCUSSION TITLE
Display calendars from the web

### DISCUSSION THREAD

**Discussion by JodliDev** [Intermesh/groupoffice#899] [Ideas]
Hello
Thank you for this great project! :)

As far as I can see, there does not seem to be a way to display calendars that exist elsewhere with groupOffice.
So for example an ics calendar that is accessible on the web or even a caldav calendar from another server.

There is the option to import ics files. But that would mean that all changes that happened after importing, would not be displayed in groupOffice.

Would that be possible to be implemented?

Thanks again for your hard work!

**Comment by mschering** [comment]
Thanks! We could build a simple module for you that periodically imports ics files. Contact us if you like.

**Reply by JodliDev** [reply]
Thank you very much for your offer. But I am using group office mainly for myself and my family. So I dont have the budget to employ you for individual plugins (as much as I want to since you deserve being paid for your work).
